.github/workflows/gh-ci.yaml

```diff
@@ -28,11 +28,17 @@ jobs:
         fail-fast: false
         matrix:
           os: [ubuntu-latest, ]
-          python-version: [3.8, 3.9, "3.10"]
+          python-version: ["3.8", "3.9", "3.10"]
           full-deps: [true, ]
           install_hole: [true, ]
           codecov: [true, ]
           include:
+            - name: py311-ubuntu-min
+              os: ubuntu-latest
+              python-version: "3.11"
+              full-deps: false
+              install_hole: true
+              codecov: false
```

Comment:
Generic question: what's the advantage of `codecov: false`? For a minimal build, wouldn't we also want to check that our code paths with missing dependencies are exercised? I don't remember if we have tests that only trigger when a dependency is missing.

Reply:
speed mostly (roughly 2x in some cases). We have a `minimal-ubuntu` runner that just down lowest python with no optional deps so I've been relying on that one being the one to pick up any "raise error if missing dependency" things.